{"uid":"fc0a963c552368d6","name":"Test Delete Admin Agent","fullName":"test_cases.test_agent.TestAgentManagement#test_delete_admin_agent","historyId":"bf5102b0fa26be747eb8ee000c1d24e7","time":{"start":1730186748527,"stop":1730186748527,"duration":0},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":true,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"autonomous_agent_api","time":{"start":1730186260565,"stop":1730186260565,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"admin_login_response","time":{"start":1730186263334,"stop":1730186263453,"duration":119},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"admin_login_cookie","time":{"start":1730186263453,"stop":1730186263453,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"create_template_fixture","time":{"start":1730186263453,"stop":1730186263575,"duration":122},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"edit_template_fixture","time":{"start":1730186263575,"stop":1730186263708,"duration":133},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"create_admin_agent_fixture","time":{"start":1730186263708,"stop":1730186263841,"duration":133},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"edit_admin_agent_fixture","time":{"start":1730186264022,"stop":1730186264210,"duration":188},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"admin_agent_fixture","time":{"start":1730186264212,"stop":1730186264399,"duration":187},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"load_funds_to_agent_address","time":{"start":1730186264400,"stop":1730186385371,"duration":120971},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"run_admin_agent_fixture","time":{"start":1730186385372,"stop":1730186747945,"duration":362573},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"delete_admin_agent_fixture","time":{"start":1730186747945,"stop":1730186748526,"duration":581},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"passed","steps":[],"attachments":[{"uid":"e8c004ba669f561d","name":"log","source":"e8c004ba669f561d.txt","type":"text/plain","size":305},{"uid":"d4bc0f9bf3633e6","name":"stdout","source":"d4bc0f9bf3633e6.txt","type":"text/plain","size":427},{"uid":"9ec1d077ca2eff53","name":"stderr","source":"9ec1d077ca2eff53.txt","type":"text/plain","size":623}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":3,"hasContent":true},"afterStages":[{"name":"autonomous_agent_api::0","time":{"start":1730186750271,"stop":1730186750271,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"feature","value":"Delete Admin Agent"},{"name":"parentSuite","value":"Agent Management"},{"name":"suite","value":"test_agent"},{"name":"subSuite","value":"TestAgentManagement"},{"name":"host","value":"fv-az1499-278"},{"name":"thread","value":"1913-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_cases.test_agent"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":15,"broken":15,"skipped":0,"passed":43,"unknown":0,"total":73},"items":[{"uid":"565c573eeed1ee1e","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/71//#testresult/565c573eeed1ee1e","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729748684509,"stop":1729748684509,"duration":0}},{"uid":"299db80cec56b429","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/70//#testresult/299db80cec56b429","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729680163006,"stop":1729680163006,"duration":0}},{"uid":"83e8c4498223241b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/69//#testresult/83e8c4498223241b","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729679365508,"stop":1729679365508,"duration":0}},{"uid":"1869d9e8c3325bc8","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/68//#testresult/1869d9e8c3325bc8","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729672197274,"stop":1729672197274,"duration":0}},{"uid":"96e4ce9af3555ce9","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/67//#testresult/96e4ce9af3555ce9","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729667682271,"stop":1729667682271,"duration":0}},{"uid":"b3cff367bfb23c07","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/66//#testresult/b3cff367bfb23c07","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729664171681,"stop":1729664171681,"duration":0}},{"uid":"670a5c6262b89a4b","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/65//#testresult/670a5c6262b89a4b","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729574248030,"stop":1729574248030,"duration":0}},{"uid":"7fb47cdaef583553","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/64//#testresult/7fb47cdaef583553","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729508364307,"stop":1729508364307,"duration":0}},{"uid":"71bf404371f81928","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/63//#testresult/71bf404371f81928","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729506658615,"stop":1729506658615,"duration":0}},{"uid":"27dc2f65bd9404c4","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/62//#testresult/27dc2f65bd9404c4","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1729245928776,"stop":1729245928776,"duration":0}},{"uid":"683580d6e2d7989d","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/61//#testresult/683580d6e2d7989d","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1728386491538,"stop":1728386491538,"duration":0}},{"uid":"21bf4d1f8e949d9a","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/60//#testresult/21bf4d1f8e949d9a","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1728380137437,"stop":1728380137437,"duration":0}},{"uid":"c2c5a81628eb1773","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/59//#testresult/c2c5a81628eb1773","status":"broken","statusDetails":"requests.exceptions.SSLError: HTTPSConnectionPool(host='api.agents.cardanoapi.io', port=443): Max retries exceeded with url: /api/auth/login (Caused by SSLError(SSLCertVerificationError(1, \"[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'api.agents.cardanoapi.io'. (_ssl.c:1000)\")))","time":{"start":1728370091631,"stop":1728370091631,"duration":0}},{"uid":"f08340b0876241eb","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/58//#testresult/f08340b0876241eb","status":"failed","statusDetails":"AssertionError: Expected POST /auth/login to succeed but got statusCode: 503 : body: <html>\r\n<head><title>503 Service Temporarily Unavailable</title></head>\r\n<body>\r\n<center><h1>503 Service Temporarily Unavailable</h1></center>\r\n<hr><center>nginx/1.23.2</center>\r\n</body>\r\n</html>","time":{"start":1728362223830,"stop":1728362223830,"duration":0}},{"uid":"3f6b0cf7fc2cec99","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/57//#testresult/3f6b0cf7fc2cec99","status":"failed","statusDetails":"AssertionError: Expected POST /auth/login to succeed but got statusCode: 503 : body: <html>\r\n<head><title>503 Service Temporarily Unavailable</title></head>\r\n<body>\r\n<center><h1>503 Service Temporarily Unavailable</h1></center>\r\n<hr><center>nginx/1.23.2</center>\r\n</body>\r\n</html>","time":{"start":1728055093143,"stop":1728055093143,"duration":0}},{"uid":"8e84407fa07982ed","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/56//#testresult/8e84407fa07982ed","status":"failed","statusDetails":"AssertionError: Expected POST /auth/login to succeed but got statusCode: 503 : body: <html>\r\n<head><title>503 Service Temporarily Unavailable</title></head>\r\n<body>\r\n<center><h1>503 Service Temporarily Unavailable</h1></center>\r\n<hr><center>nginx/1.23.2</center>\r\n</body>\r\n</html>","time":{"start":1728054163824,"stop":1728054163824,"duration":0}},{"uid":"6baf2086972932f2","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/55//#testresult/6baf2086972932f2","status":"failed","statusDetails":"AssertionError: Expected POST /auth/login to succeed but got statusCode: 503 : body: <html>\r\n<head><title>503 Service Temporarily Unavailable</title></head>\r\n<body>\r\n<center><h1>503 Service Temporarily Unavailable</h1></center>\r\n<hr><center>nginx/1.23.2</center>\r\n</body>\r\n</html>","time":{"start":1728031538209,"stop":1728031538209,"duration":0}},{"uid":"55b00150fbe6f3be","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/54//#testresult/55b00150fbe6f3be","status":"failed","statusDetails":"AssertionError: Expected POST /auth/login to succeed but got statusCode: 503 : body: <html>\r\n<head><title>503 Service Temporarily Unavailable</title></head>\r\n<body>\r\n<center><h1>503 Service Temporarily Unavailable</h1></center>\r\n<hr><center>nginx/1.23.2</center>\r\n</body>\r\n</html>","time":{"start":1728022312778,"stop":1728022312778,"duration":0}},{"uid":"c11587892a0c8e4c","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/53//#testresult/c11587892a0c8e4c","status":"broken","statusDetails":"requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","time":{"start":1728020022693,"stop":1728020022693,"duration":0}},{"uid":"4e0d57835c738ccb","reportUrl":"https://cardanoapi.github.io/autonomous-agents/test_backend_allure_report/52//#testresult/4e0d57835c738ccb","status":"broken","statusDetails":"requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","time":{"start":1728019910791,"stop":1728019910791,"duration":0}}]},"tags":[]},"source":"fc0a963c552368d6.json","parameterValues":[]}